Sources/Public/iOS/Compatibility/CompatibleAnimationView.swift

```diff
@@ -187,6 +187,15 @@ public final class CompatibleAnimationView: UIView {
     }
   }
 
+  @objc
+  public var compatibleAnimationTextProvider: CompatibleAnimationTextProvider? {
+    didSet {
+      if let textProvider = compatibleAnimationTextProvider?.textProvider {
+        animationView.textProvider = textProvider
+      }
```

Comment:
We probably want to handle the case where `compatibleAnimationTextProvider` is set to `nil`. Previously this setter did nothing in that case, so `animationView.textProvider` would still have the previous value.

```suggestion
      animationView.textProvider = compatibleAnimationTextProvider?.textProvider ?? DefaultTextProvider()
```